charts/rapid-response/templates/_helpers.tpl

```diff
@@ -170,4 +170,18 @@ Determine collector endpoint based on provided region or .Values.rapidResponse.a
     {{- else if (eq .Values.global.sysdig.region "au1") -}}
         {{- "app.au1.sysdig.com" -}}
     {{- end -}}
+{{- end -}}
+
+{{/*
+Rapid Response have the environment variable skip_tls_check: true for skip the certficate verification
+while we do the other way round for our other components (sslVerifyCertificate: false for disabling the check).
+The aim of rapidResponse.certificateValidation is to align the settings with the other Sysdig charts,
+without introducing changes on Rapid Response container image.
+*/}}
+{{- define "rapidResponse.certificateValidation" -}}
+    {{- if or ( eq (.Values.rapidResponse.skipTlsVerifyCertificate | toString) "true") (eq (.Values.rapidResponse.sslVerifyCertificate | toString) "false") -}}
```

Comment:
you don't need toString and compare it with a string.

Reply:
So you mean creating an if like that:
```
{{- if or (.Values.rapidResponse.skipTlsVerifyCertificate) (.Values.rapidResponse.sslVerifyCertificate) -}}
```
but how I can specify `Values.rapidResponse.sslVerifyCertificate` as false?

Reply:
not Values.rapidResponse.sslVerifyCertificate